pipelineruns/notebooks/.tekton/odh-workbench-codeserver-datascience-cpu-py312-v2-25-push.yaml

```diff
@@ -22,6 +22,8 @@ metadata:
   name: odh-workbench-codeserver-datascience-cpu-py312-v2-25-on-push
   namespace: rhoai-tenant
 spec:
+  timeouts:
+    pipeline: 4h
```

Comment:
if you're sure this is the right timeout... then ok

i worry we hit the 2 hour timeout for pipeline step, and i believe 2h for step is maximum and cannot be prolonged